src/com/session/Check.java

```diff
@@ -24,18 +31,29 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOExc
 
 	public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
 		resp.setContentType("text/html");
-		PrintWriter o = resp.getWriter();
+		int count = 0;
 		String email = req.getParameter("email");
 		String password = req.getParameter("password");
-
-		if ((email.equals("admin@testing.com")) && (password.equals("123"))){
+		Query q1 = new Query("Contact");
```

Comment:
again same mistake, fetching all contact and then checking each one manually is totally not the right way